src/SourceConstraints.hs

```diff
@@ -115,31 +119,29 @@ warnings dynFlags (L sourceSpan node) =
         neverQualify
 
     descend :: Data a => a -> WarningMessages
-    descend =
-      unionManyBags . gmapQ
-        (descend `ext2Q` warnings dynFlags)
+    descend = unionManyBags . gmapQ (descend `ext2Q` warnings context)
```

Comment:
@mbj This hunk is unrelated to the `Change` and should be extracted to a prefixed `Refactor` (or formatting `Fix`)